src/Adapter/MSTest.TestAdapter/Execution/TestClassInfo.cs

```diff
@@ -302,6 +303,8 @@ public void RunClassInitialize(TestContext testContext)
                     finally
                     {
                         IsClassInitializeExecuted = true;
+                        Debug.Assert(testContext is ITestContext, "TestContext is null.");
```

Comment:
Test context is not null if the assertion fails, it means that test context doesn't implement `ITestContext` :)